third_party/BUILD

```diff
@@ -1,4 +1,5 @@
-load("@rules_java//java:defs.bzl", "java_import", "java_library", "java_plugin")
+load("@rules_java//java:defs.bzl", "java_library", "java_plugin")
+load("//tools/distributions:distribution_rules.bzl", java_import = "distrib_java_import")
```

Comment:
Overwriting a well known symbol with another one seems hacky. Why not make it explicit where you need it?

Reply:
I try to be as less intrusive for Bazel developer as possible. Because most Bazel developer don't have to know what distrib_java_import is. But I don't have a strong opinion on this. @philwo WDYT?

Reply:
I agree with Tobi - it's probably more confusing if a well-known rule like `java_import` suddenly acts differently. I'd make it explicit and just call it `distrib_java_import`.

Reply:
Cool, it's done!